src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -147,11 +157,16 @@ const AccessibilityRequestDetailPage = () => {
           shouldSendEmail: values.shouldSendEmail
         }
       }
-    }).then(() => {
-      refetch();
-      showMessage(''); // allows screen reader to hear consecutive success message
-      showMessage(t('requestDetails.notes.confirmation', { requestName }));
-      resetForm({});
+    }).then(response => {
+      const userErrors =
+        response.data?.createAccessibilityRequestNote?.userErrors;
+      if (!userErrors) {
+        refetch();
+        showMessage(''); // allows screen reader to hear consecutive success message
```

Comment:
Is this for VO? It has bugs in its alert announcing that are pretty distinct from what we hear in JAWS. We probably need to test this in JAWS if there are issues that need to be worked around.

If this is something we need to always do in VO, we should see if we can bake it into the message announcing code.

Reply:
The issue is that it was not re-rendering with successive successes, so it was not announcing.  This was only tested in VO.  I'm curious what the bugs are.